### ISSUE TITLE
[BUG] Idea Manual-Proxy Exception 

### ISSUE THREAD

**Issue by a222757** [Exafunction/codeium#66]
I encountered with following problem when using codeium in Idea(win11), when I use manual httpProxy
`Unhandled exception in [StandaloneCoroutine{Cancelling}@20269868, Dispatchers.IO]`

log:

```
Unhandled exception in [StandaloneCoroutine{Cancelling}@20269868, Dispatchers.IO]

com.intellij.execution.IllegalEnvVarException: Illegal value of environment variable value 'NO_PROXY': 'null'
	at com.intellij.execution.configurations.GeneralCommandLine.validateAndPrepareCommandLineForLocalRun(GeneralCommandLine.java:420)
	at com.intellij.execution.configurations.GeneralCommandLine.createProcess(GeneralCommandLine.java:350)
	at com.intellij.execution.process.OSProcessHandler.startProcess(OSProcessHandler.java:86)
	at com.intellij.execution.process.OSProcessHandler.<init>(OSProcessHandler.java:46)
	at com.intellij.execution.process.KillableProcessHandler.<init>(KillableProcessHandler.java:41)
	at com.codeium.intellij.language_server.LanguageServerProcessHandler.<init>(LanguageServerProcessHandler.kt:18)
	at com.codeium.intellij.language_server.LanguageServerLauncherActivity$runActivity$1.invokeSuspend(LanguageServerLauncherActivity.kt:394)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.internal.ScopeCoroutine.afterResume(Scopes.kt:32)
	at kotlinx.coroutines.AbstractCoroutine.resumeWith(AbstractCoroutine.kt:102)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:46)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:108)
	at kotlinx.coroutines.internal.LimitedDispatcher$Worker.run(LimitedDispatcher.kt:115)
	at kotlinx.coroutines.scheduling.TaskImpl.run(Tasks.kt:103)
	at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:584)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:793)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:697)
	at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:684)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelled}@20269868, Dispatchers.IO]

```

When I use No Proxy or Auto-detect will not occur this problem

**Comment by a222757** [comment]
At codeium version 1.8.58, No error exists in previous version.

**Comment by a222757** [comment]
Found fixed in version 1.8.66
